include/fizzy/fizzy.h

```diff
@@ -90,6 +90,14 @@ FizzyInstance* fizzy_instantiate(const FizzyModule* module,
 /// Free resources associated with the instance.
 void fizzy_free_instance(FizzyInstance* instance);
 
+/// Get pointer to memory of an instance.
+/// @returns Pointer to memory data or NULL in case instance doesn't have any memory.
```

Comment:
"Any" or "any exported" memory?

Reply:
It gives access to any memory

Reply:
Can you clarify that in the description?

Reply:
Added a note.